packages/firebase_crashlytics/lib/src/firebase_crashlytics.dart

```diff
@@ -0,0 +1,185 @@
+// Copyright 2019, the Flutter project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+part of firebase_crashlytics;
+
+/// The entry point for accessing Crashlytics.
+///
+/// You can get an instance by calling `Crashlytics.instance`.
+class Crashlytics {
+  static final Crashlytics instance = Crashlytics();
+
+  /// Set to true to have Errors sent to Crashlytics while in debug mode.
+  bool reportInDevMode = false;
+
+  /// Keys to be included with report.
+  @visibleForTesting
+  final Map<String, dynamic> keys = <String, dynamic>{};
+
+  /// Logs to be included with report.
+  @visibleForTesting
+  final ListQueue<String> logs = ListQueue<String>(15);
+  int logSize = 0;
+
+  bool get isInDebugMode {
```

Comment:
I don't think this needs to be part of the public API. We might want to change it later. Since it's only used in one place, you can just move the logic there.

Reply:
Done